Cargo.toml

```diff
@@ -1,13 +1,15 @@
 [package]
 name = "threshold_crypto"
 # REMINDER: Update version in `README.md` when incrementing:
-version = "0.1.0"
-authors = ["Vladimir Komendantskiy <komendantsky@gmail.com>",
-           "Andreas Fackler <AndreasFackler@gmx.de>",
-           "Peter van Nostrand <jnz@riseup.net>",
-           "Andrew Gross <andogro@gmail.com>",
-           "Nick Sanders <nsan1129@gmail.com>",
-           "Marc Brinkmann <git@marcbrinkmann.de>"]
+version = "0.2.0"
```

Comment:
The style guide also asks to sort keys alphabetically within each section with very special rules for this section. I understood the order as follows: "name", "version", all the remaining keys but "description" in alphabetical order, and then "description".

Reply:
You're right. I amended the commit.